### DISCUSSION TITLE
Lumen: Errors not shown (The server returned a "500 Internal Server Error".)

### DISCUSSION THREAD

**Discussion by simplenotezy** [laravel/framework#38589] [Q&A]
I am playing around with Lumen for the first time, after having used Laravel for a long time. My issue is that errors are not shown. Instead I see this error page:

![image](https://user-images.githubusercontent.com/3786627/131265915-e6e7f8cb-d26e-4b50-a625-7a34d1123620.png)

I am using Laravel Valet to run lumen on my local machine.

I have verified that APP_DEBUG is set to true.

If I go into `storage/logs` I see all the errors there, but I wonder why they are not shown in the browser.

Any ideas?

**Comment by talelmishali** [comment]
What about the ```APP_ENV``` ?

**Reply by simplenotezy** [reply]
Thanks for your answer!

They should both be correct:
```
APP_ENV=local
APP_DEBUG=true
```
